### PR TITLE
WT-7909 Use a new method to check concurrent user transactions before starting rollback to stable

### PR DESCRIPTION
**Pull request by hbokhari**

### CODE REVIEW

File: src/txn/txn.c [hunk 1]
```diff
@@ -138,6 +138,47 @@ __wt_txn_release_snapshot(WT_SESSION_IMPL *session)
...
+    /*
+     * No lock is required because the session array is fixed size, but it may contain inactive
+     * entries. We must review any active session, so insert a read barrier after reading the active
+     * session count. That way, no matter what sessions come or go, we'll check the slots for all of
+     * the user sessions for active transactions when we started our check.
+     */
+    WT_ORDERED_READ(session_cnt, conn->session_cnt);
+    for (i = 0, session_in_list = conn->sessions; i < session_cnt; i++, session_in_list++) {
+        /* Skip checking inactive sessions. */
+        if (!session_in_list->active)
```
Comment (hbokhari) [inline src/txn/txn.c:167]: @kommiharibabu, the only change from last time you reviewed this PR is the check for inactive sessions.

Comment (kommiharibabu) [reply src/txn/txn.c:167]: Instead of the additional word of `checking`, How about `Skip inactive sessions`?

**Review by kommiharibabu — APPROVED** [review]
LGTM

File: src/txn/txn.c (hunk 1 above)
Comment (hbokhari) [reply src/txn/txn.c:167]: Fixed.

**Pull request merged**